### DISCUSSION TITLE
Various error on fresh install

### DISCUSSION THREAD

**Discussion by hmorandell** [photoview/photoview#192] [Q&A and Troubleshooting] [answered]
Hi,
I have set up photoview using the dockerfile as a start point, changed it like this:

```
      - PHOTOVIEW_PUBLIC_ENDPOINT=http://<PUBLIC_IP>:8000/
      -   - ./photos:/home/<MYUSER>/photos:ro
```

I kept the rest the same, everything works fine, I can setup the admin user, I can login, I have photos in the photo folder but when I click "Scan all" I receive no error, but clicking on Albums or Photos I get:

**Something went wrong
Server error: could not query thumbnail (): record not found at (myAlbums,0,thumbnail,thumbnail)**

**Albums
Error could not query thumbnail (): record not found**

There are no errors in the logfile:

> 2021/02/12 10:27:10 Queue waiting for lock,
> 2021/02/12 10:27:10 Queue running: in_progress: 0, max_tasks: 3, queue_len: 1,
> 2021/02/12 10:27:10 Queue starting job,
> 2021/02/12 10:27:10 Broadcasting notification: Scanning media,
> 2021/02/12 10:27:10 Queue waiting,
> 2021/02/12 10:27:10 Starting job,
> 2021/02/12 10:27:10 POST 200 xxxxxxxxxxxxxx:8000/api/graphql 9.671663ms user: admin,
> 2021/02/12 10:27:10 Job finished,
> 2021/02/12 10:27:10 Queue waiting for lock,
> 2021/02/12 10:27:10 Queue running: in_progress: 0, max_tasks: 3, queue_len: 0,
> 2021/02/12 10:27:10 Broadcasting notification: Scanner complete,
> 2021/02/12 10:27:10 Queue waiting,
> 2021/02/12 10:27:10 POST 200 xxxxxxxxxxxxx:8000/api/graphql 9.568278ms user: admin,
> 2021/02/12 10:27:15 POST 200 xxxxxxxxxxxx:8000/api/graphql 4.280166ms user: admin,
> 

Anyone a hint what I'm missing?

thanks!

**Comment by viktorstrate** [comment, marked as answer]
Hi @hmorandell this happends if the scanner has scanned a photo or video only partially.
I have actually fixed this already and it will be published with the next release.

If your scanner is finished and you still experience this, you can try the `viktorstrate/photoview:edge` docker build instead until the next version is released.

**Comment by hmorandell** [comment]
great! it seems to work now!
thanks for the quick reply!
